eclass/kernel-install.eclass

```diff
@@ -549,6 +549,25 @@ kernel-install_pkg_pretend() {
 	fi
 }
 
+# @FUNCTION: kernel-install_pkg_pretend
+# @DESCRIPTION:
+# Print a warning if the initramfs use flag is not set
+kernel-install_pkg_setup() {
+	if ! use initramfs; then
```

Comment:
I would perhaps only do it on new installs and/or first transition USE=initramfs->USE=-initramfs?

Reply:
Good idea, fixed!